src/patternfly/components/Touchspin/touchspin.scss

```diff
@@ -0,0 +1,43 @@
+.pf-c-touchspin {
+  // main
+  --pf-c-touchspin__main--m-active--c-form-control--Color: var(--pf-global--Color--100);
+
+  // units
+  --pf-c-touchspin__main--c-input-group--MarginLeft: var(--pf-global--spacer--sm);
+
+  // control
+  --pf-c-touchspin__control--m-threshold--c-button--Color: var(--pf-global--disabled-color--100);
+
+  // icon
+  --pf-c-touchspin__icon--FontSize: var(--pf-global--FontSize--xs);
+
+  // form control
+  --pf-c-touchspin--c-form-control--MaxWidth: #{pf-size-prem(70px)};
```

Comment:
wdyt about using `size` instead of CSS `max-width`, and showing examples with `size` updated depending on the number of characters in the input - implying react would update it on the fly as the value changes? React could also have a `size` prop that sets it to something custom as a way to manage the width.

Reply:
could you show me an example of using size. is it a property ? or a concept for the var name ?

Reply:
https://codepen.io/mcoker/pen/WNxoxeq